ArchUnitNET/Fluent/Syntax/Elements/Types/Classes/ClassConditionsDefinition.cs

```diff
@@ -23,6 +24,11 @@ public static ICondition<Class> BeSealed()
                 "is not sealed");
         }
 
+        public static ICondition<Class> BeImmutable()
+        {
+            return new SimpleCondition<Class>(cls => cls.Members.All(m => m.IsImmutable != false), "be immutable",
+                "is not immutable");
```

Comment:
In my own implementation, I ignored static fields (which thus may remain mutable), because I only care about instance immutability. This applies to all checks here.

Reply:
Good point, I forgot about statics. Thanks for pointing it out.